src/main/java/com/mathworks/ci/RunMatlabCommandBuilder.java

```diff
@@ -96,21 +96,21 @@ public void perform(@Nonnull Run<?, ?> build, @Nonnull FilePath workspace,
             @Nonnull Launcher launcher, @Nonnull TaskListener listener)
             throws InterruptedException, IOException {
 
-        // Set the environment variable specific to the this build
-        setEnv(build.getEnvironment(listener));
+        // Get the environment variable specific to the this build
+        final EnvVars env = build.getEnvironment(listener);

```

Comment:
Is the class-level field `env` used anymore? If not, can it be removed along with `setEnv`?

Reply:
Done.